lib/manageiq/network_discovery.rb

```diff
@@ -79,11 +91,20 @@ def self.scanHost(sInfo)
       # and skip scanning if the ping fails.
       pingOk = true
       begin
-          pingOk = Net::Ping::External.new(sInfo.ipaddr).ping if sInfo.usePing
-        rescue Timeout::Error
-          pingOk = false
+        pingOk = Net::Ping::External.new(sInfo.ipaddr).ping if sInfo.usePing
+      rescue Timeout::Error
+        pingOk = false
+      end
+
+      if pingOk
+        # Trigger probes
+        sInfo.discover_types.each do |type|
+          klass = Object.const_get(PROVIDERS_BY_TYPE[type])
+          $log.info "#{klass}: probing ip = #{sInfo.ipaddr}" if $log
+          klass.send(:probe, sInfo)
```

Comment:
Minor, but this can just be `klass.probe(sInfo)`

Reply:
I missed that. Thank anyway!